installers/pgo-installer/ansible/roles/pgo-preflight/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+---
+- fail:
+    msg: "Please specify the a tag: uninstall, uninstall-metrics, install, install-metrics or update"
```

Comment:
May need to pull out the metrics options and update the install/uninstall/upgrade tags